src/clr-core/internal/utils/string.ts

```diff
@@ -26,6 +26,18 @@ export function camelCaseToKebabCase(value: string) {
   return value.replace(/[A-Z]/g, l => `-${l.toLowerCase()}`);
 }
 
+export function kebabCaseToCamelCase(str: string) {
+  return str
+    .split('-')
+    .map((item, index) => (index ? item.charAt(0).toUpperCase() + item.slice(1).toLowerCase() : item))
+    .join('');
+}
+
+export function kebabCaseToPascalCase(string: string) {
+  const camelCase = kebabCaseToCamelCase(string);
+  return camelCase[0].toUpperCase() + camelCase.substr(1);
```

Comment:
This line is logically equivalent to `capitalizeFirstLetter` function below. Curious if there's a reason for using `substr` in this one and `slice` in the other one?

Reply:
Updated to use the same function